pepper-apis/src/main/java/org/broadinstitute/ddp/model/copy/CopyConfigurationPair.java

```diff
@@ -0,0 +1,57 @@
+package org.broadinstitute.ddp.model.copy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CopyConfigurationPair {
+
+    private long id;
+    private CopyLocation source;
+    private CopyLocation target;
+    private List<CompositeCopyConfigurationPair> compositeChildLocations = new ArrayList<>();
+    private int order;
```

Comment:
The `order` is used when inserting the pairs.

Reply:
Probably not. But since it's a list and the study staff specified the configuration in a certain order, we're preserving and executing based on that order. Also, I think I like to have this class carry around the `order` property -- it more fully reflect the data in database and gives you something to order by if other code wants to do other orderings.